config/routes.rb

```diff
@@ -1,4 +1,5 @@
 require "resque_web"
+ResqueWeb::Engine.eager_load!
```

Comment:
I kept getting intermittent errors in dev when I navigated to /resque_web.  I searched google for solutions and it gave this as its answer.  I sort of understand what this line is doing, but if you think that it would work better later on in the file then I can move it.

Where would it go?  After the ResqueWeb mount, or after the final end?